### DISCUSSION TITLE
Failed to load Config

### DISCUSSION THREAD

**Discussion by Spellarj123** [saadeghi/daisyui#2920] [Q&A]
I have a laravel project using tailwindcss and daisyui. When I run 'npm run dev', an error shows up

failed to load config from C:\xampp\htdocs\ChicStyle\vite.config.js
error when starting dev server:
Error: Dynamic require of "file:///C:/xampp/htdocs/ChicStyle/node_modules/daisyui/src/index.js" is not supported
    at file:///C:/xampp/htdocs/ChicStyle/vite.config.js.timestamp-1710817309146-3b861de2b1115.mjs:6:9
    at file:///C:/xampp/htdocs/ChicStyle/vite.config.js.timestamp-1710817309146-3b861de2b1115.mjs:68:13
    at ModuleJob.run (node:internal/modules/esm/module_job:193:25)
    
When I remove the plugins: [require("daisyui")] in tailwindcss.config.js and run, the project works fine. If I return the plugin, the project works fine but shows the following lines
[vite] tailwind.config.js changed, restarting server...
failed to load config from C:\xampp\htdocs\ChicStyle\vite.config.js
vite] Dynamic require of "file:///C:/xampp/htdocs/ChicStyle/node_modules/daisyui/src/index.js" is not supported
[vite] server restart failed

**Comment by saadeghi** [comment]
If you're using ESBuild, it might be because of this issue:  
https://github.com/evanw/esbuild/issues/3324
